R/model.R

```diff
@@ -156,7 +156,19 @@ parse_value <- function(info, value, disable_date_parsing = FALSE) {
   } else if (info$type == "bool") {
     return(as.logical(value))
   } else if (info$type == "int") {
-    return(as.integer(value))
+    # Int doesn't have enough capacity to store some weekly `pub_wiki` values.
+    value <- as.double(value)
+    if (any(value != round(value))) {
+      cli::cli_warn(
+        c(
+          "Values in {info$name} were expected to be integers but contain a decimal component",
+          "i" = "Decimal components are returned as-is"
+        ),
+        class = "epidatr__int_nonzero_decimal_digits"
+      )
+    }
+
```

Comment:
Could use this: https://mllg.github.io/checkmate/reference/checkIntegerish.html

Reply:
Thanks, that's cleaner. Unfortunately, their check is too thorough -- it also fails when values are too large to be stored as ints, which is the case here.

Reply:
Oof, too bad